src/test/java/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,153 @@
+package subway.acceptance;
+
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.acceptance.LineRequestFixture.*;
+import static subway.acceptance.StationRequestFixture.지하철_역_생성;
+
+@DisplayName("지하철 노선 관련 기능")
+public class LineAcceptanceTest extends AcceptanceTest {
+
+    /**
+     * Given 2개의 지하철 역을 생성하고
+     * When (해당 지하철 역을 포함하는) 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선을 생성 한다.")
+    @Test
+    void createLine() {
+        // given
+        Long upStationId = 지하철_역_생성("강남역").jsonPath().getLong("id");
+        Long downStationId = 지하철_역_생성("양재역").jsonPath().getLong("id");
+
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_생성("신분당선", "bg-red-600", upStationId, downStationId, 10L);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
```suggestion
        // when
        지하철_노선_생성("신분당선", "bg-red-600", upStationId, downStationId, 10L);
```

204에 대한 검증은 지워줘도 괜찮을 것 같네요.